src/main/java/ch/naviqore/service/DepartureTime.java

```diff
@@ -0,0 +1,9 @@
+package ch.naviqore.service;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.time.LocalDate;
+
+public interface DepartureTime {
+    @NotNull LocalDate getDepartureTime();
```

Comment:
Again, I would not wrap a single data type. We can achieve clarity here by using the variable name "departueTime". Or is there a benefit I don't see yet?